src/api/users/src/models/schema.js

```diff
@@ -16,6 +16,34 @@ const validateId = () =>
     },
   });
 
+// no user can be created as an admin by default
+const validateUserRights = () => (req, res, next) => {
+  req.body.isAdmin = false;
+  next();
+};
+
+const updateUser = (makeAdmin = false) => async (req, res, next) => {
+  const { id } = req.params;
+  const { body } = req;
+
+  try {
+    const userRef = db.doc(id);
+    const doc = await userRef.get();
+
+    if (!doc.exists) {
+      next(createError(404, `user ${id} not found.`));
+    } else {
+      // if makeAdmin is true turn the user into an admin, if not use body to update user.
+      const user = new User(makeAdmin ? { ...body, isAdmin: true } : body);
```

Comment:
This won't allow downgrading a user from Admin to normal, which we should support.  We should accept a value `{isAdmin: true|false}` on the body and use that instead of hard-coding here.